packages/server/modules/comments/tests/comments.graph.spec.js

```diff
@@ -1,4 +1,853 @@
-describe('Graphqql @comments', () => {
+const expect = require('chai').expect
```

Comment:
Didn't go through this very thoroughly, but it's cool that you're running operations against `apollo`, not the service layer directly, and that it's all defined so abstractly

Reply:
The point of this is, that after all this boilerplate, it was and will be really easy to add user, stream, action permutations to the test data object and all the rest is done automatically.